app/src/main/java/com/example/event_finder_capstone/LoginActivity.java

```diff
@@ -28,24 +22,14 @@ protected void onCreate(Bundle savedInstanceState) {
 
         etUsername = findViewById(R.id.etUsername);
         etPassword = findViewById(R.id.etPassword);
-        btnLogin = findViewById(R.id.btnLogin);
-        btnSignUp = findViewById(R.id.btnSignUp);
+        Button btnLogin = findViewById(R.id.btnLogin);
+        Button btnSignUp = findViewById(R.id.btnSignUp);
 
-        //checkIfLoggedIn();
+        checkIfLoggedIn();
 
-        btnLogin.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                loginUser();
-            }
-        });
+        btnLogin.setOnClickListener(v -> loginUser());
```

Comment:
💡 FYI:  I assume these were changes were suggested by Android Studio.  If you're not familiar with the syntax, you can read up more here: https://docs.oracle.com/javase/tutorial/java/javaOO/lambdaexpressions.html

Reply:
Assumed correctly - was actually planning to ask about that during our 1:1 so thank you for the resource!